radixdlt-core/docker/scripts/fix-vulnerabilities.sh

```diff
@@ -1,19 +1,3 @@
-#Update curl https://curl.se/docs/CVE-2021-22876.html
```

Comment:
Ubuntu 22.04 is shipped with `curl 7.81.0 libcurl/7.81.0 OpenSSL/3.0.2 zlib/1.2.11 brotli/1.0.9 zstd/1.4.8 libidn2/2.3.2 libpsl/0.21.0 (+libidn2/2.3.2) libssh/0.9.6/openssl/zlib nghttp2/1.43.0 librtmp/2.3 OpenLDAP/2.5.12` so it is safe to remove this now-legacy hard dependency on curl 7.76.1.